rosidl_typesupport_introspection_cpp/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ ament_python_install_package(${PROJECT_NAME})
 add_library(${PROJECT_NAME} SHARED src/identifier.cpp)
 if(WIN32)
   target_compile_definitions(${PROJECT_NAME}
-    PRIVATE "ROSIDL_TYPESUPPORT_INTROSPECTION_CPP_BUILDING_DLL")
+    PRIVATE "-DROSIDL_TYPESUPPORT_INTROSPECTION_CPP_BUILDING_DLL")
```

Comment:
I would avoid the `-D` prefix which is not necessary. Imo without is the more readable variant.